cyto_dl/models/im2im/utils/postprocessing/auto_thresh.py

```diff
@@ -1,9 +1,9 @@
 import importlib
-from typing import Union
+from typing import Union, Optional
 
 
 class AutoThreshold:
-    def __init__(self, method: Union[float, str]):
+    def __init__(self, method: Optional[Union[float, str]] = None):
```

Comment:
Thanks for making these changes! For future reference, in order to pass `method=None`, what would the override look like? Something like `CytoDLModel.override_config({"...method": None, ...})`?

Reply:
yep, that should be fine!